pyre/pyrex/tests/fixtures/cpyext_callables.c

```
@@ -0,0 +1,202 @@
+    method = PyMethod_New(function, receiver);
+    if (method == NULL) {
+        return NULL;
+    }
+    answer = Py_BuildValue("OiOO", method, PyMethod_Check(method),
+                           PyMethod_GET_FUNCTION(method), PyMethod_GET_SELF(method));
+    Py_DECREF(method);
+    return answer;
+}
+
+/* `sys.modules`, which an extension reaches without importing `sys`. */
+static PyObject *module_dict(PyObject *self, PyObject *unused)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'module_dict' of similar type ('PyObject *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)